src/js/schemas/service-schema/Labels.js

```diff
@@ -9,6 +9,10 @@ const Labels = {
       getter: function (service) {
         let labels = service.getLabels() || {};
 
+        if (Object.keys(labels).length === 0) {
+          return null;
```

Comment:
Changed this problem at the root, so if now a array is provided with a length === 0 the default row is created otherwise the provided array is used.